src/main.cpp

```diff
@@ -249,6 +250,7 @@ boolean setupDone = false;
 int backflushON = 0;             // 1 = backflush mode active
 int flushCycles = 0;             // number of active flush cycles
 int backflushState = 10;         // counter for state machine
+boolean waterFull = true;               // current state of water reservoir, true=full, false=empty
```

Comment:
The comment is a bit redundant, I think it's pretty clear from context what this variable is for.

Reply:
fixed and added that without water sensor this will stay on "true"